website/docs/form/switch.mdx

```diff
@@ -32,8 +32,10 @@ import { Switch } from "@chakra-ui/core"
 
 ```jsx
 <Flex justify="center" align="center">
```

Comment:
Can we move the flex styles to `FormControl` here

Reply:
I'm noticing something interesting here. I've switched it to `FormControl as={Flex}` in order to enable flex props on `FormControl`, but if I use `justify` or `align` it's ignoring those props. I have to use `justifyContent` and `alignItems` to make it work correctly. Is that expected?